cli/cmd/agent_aws-install_ec2ic_test.go

```diff
@@ -30,6 +31,7 @@ func TestAwsInstallEC2IC(t *testing.T) {
 		t.Skip("aws credentials not found in environment, skipping test")
 	}
 
+	cli.Log = lwlogger.New("DEBUG").Sugar()
```

Comment:
Is this intentional? Why do we hard code this?

Reply:
Done with 61813be7983295cc225feb943a0722b6f5276763